packages/gestalt/src/utils/datavizcolors/datavizcolors.test.js

```diff
@@ -0,0 +1,105 @@
+// @flow strict
+import getCheckboxColor from './getCheckboxColor.js';
```

Comment:
Feel free to put `lightModeTheme` in a shared `fixtures` file, but this file should be broken up into separate test files for each util tested. There should always be a 1:1 correlation of `file.js`:`file.test.js`. This makes it easy to see what has tests and what doesn't, and to identify dead tests when removing files.

Reply:
fixed the naming. Didn't add to fixtures in the PR, but we can do it at a different time